### PR TITLE
Hari/feature/homepage

### PR DESCRIPTION
**Pull request by balaharikrishna**
Created Homepage for desktop view and added HTML ,CSS,SCSS files to the branch "Hari/feature/homepage".
Majorly added Home and OurServices html Pages.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+       </div>
+                                  <!-- MainContentGrid2 -->
+        <div class="MainContentGrid">
+            <img src="./Images/tile2L.jpg" class="tileimages" alt="logo">
+            <div>
+              <p class="MaincontentPara">
+                FTZ World Services is your ticket to the world.We're the 
+                licensed operator of the Miami Free Trade Zone, the "Gateway to the Americas."
+                We process as much as $1 billion in goods from more than 70 countries each year.
+                 <br><br>
```
Comment (akashdoppalapudi) [inline index.html]: don't use <br> tags...
use css styling instead in all places

File: ourservicesstyles.css [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+  color: white;
+}
+
+.GridNavSection {
+  display: grid;
+  grid-template-columns: 25% 75%;
+  background-color: rgba(1, 15, 29, 0.8);
+  border: 1px solid rgb(23, 45, 69);
+  border-style: ridge;
+  outline-color: 0px;
```
Comment (akashdoppalapudi) [inline ourservicesstyles.css]: color in pixels?

File: ourservicesstyles.css [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+  border-right: 1px solid white;
+  padding: 0 2% 0 2%;
+  font-size: 55%;
+}
+.Grid-TopRightMenu .TopMenus .TopMenus-span .TopMenusLink {
+  text-decoration: none;
+  color: white;
+}
+.Grid-TopRightMenu .TopMenus .TopMenus-span1 {
+  padding: 0 0 0 2%;
```
Comment (akashdoppalapudi) [inline ourservicesstyles.css]: use padding-left if you're not going to use any more paddings

File: ourservicesstyles.scss [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+        right: 0;
+        background-color: rgba(10, 44, 68, 0.5);
+        color: white;
+        width: 230px;
+        height: 25px;
+        cursor: pointer;
+
+        .TopMenus-span {
+            border-right: 1px solid white;
+            padding: 0 2% 0 2%;
```
Comment (akashdoppalapudi) [inline ourservicesstyles.scss]: padding: 0 2%;

File: ourservicesstyles.scss [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+    grid-template-columns: 30% 70%;
+    background-color: rgba(1, 15, 29, 0.8);
+    border: 1px solid rgba(23, 45, 69);
+    border-style: ridge;
+    outline-color: 0px;
+    outline: 1px solid rgba(178, 189, 181, 0.2);
+    outline-offset: -5px;
+    padding: 15px;
+    grid-gap: 15px;
+    margin: 25px 0 0 0;
```
Comment (akashdoppalapudi) [inline ourservicesstyles.scss]: margin-top

File: ourservicesstyles.scss [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+
+    .MainContentGrid1 {
+        display: flex;
+        flex-direction: column;
+        padding: 10px;
+
+        .tileimages {
+            padding: 0 0 10px 0;
```
Comment (akashdoppalapudi) [inline ourservicesstyles.scss]: padding-bottom

File: ourservicesstyles.scss [hunk 7]
```diff
@@ -0,0 +1,271 @@
...
+            display: flex;
+            flex-direction: column;
+            padding: 10px 0 10px 0;
+            border-top: 1.5px solid #404649;
+            border-bottom: 1.5px solid #404649;
+        }
+    }
+
+    .MainContentGrid2 {
+        margin: 0 5% 0 0;
```
Comment (akashdoppalapudi) [inline ourservicesstyles.scss]: same here

File: ourservicesstyles.scss [hunk 8]
```diff
@@ -0,0 +1,271 @@
...
+    }
+
+    .MainContentInnerGrids {
+        display: grid;
+        grid-template-rows: auto auto auto auto;
+
+        .MainHeading {
+            .heading {
+                color: rgb(148, 193, 220);
+                margin: 0px 0 0px 0;
```
Comment (akashdoppalapudi) [inline ourservicesstyles.scss]: margin: 0

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+.background-image {
+  background: url("./Images/bg.jpg") no-repeat center center fixed;
+  background-size: cover;
+}
+
+.Grid-TopRightMenu {
+  display: grid;
+  grid-template-columns: auto;
+  position: relative;
+  padding: 0 0 45px 0;
```
Comment (akashdoppalapudi) [inline styles.css]: padding-bottom

File: styles.scss [hunk 10]
```diff
@@ -0,0 +1,201 @@
...
+    -webkit-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+}
+
+.Grid-TopRightMenu {
+    display: grid;
+    grid-template-columns: auto;
+    position: relative;
+    padding: 0 0 45px 0;
```
Comment (akashdoppalapudi) [inline styles.scss]: padding-bottom

File: styles.scss [hunk 11]
```diff
@@ -0,0 +1,201 @@
...
+        right: 0;
+        background-color: rgba(10, 44, 68, 0.5);
+        color: white;
+        width: 230px;
+        height: 25px;
+        cursor: pointer;
+
+        .TopMenus-span {
+            border-right: 1px solid white;
+            padding: 0 2% 0 2%;
```
Comment (akashdoppalapudi) [inline styles.scss]: padding: 0 2%;

File: styles.scss [hunk 12]
```diff
@@ -0,0 +1,201 @@
...
+.Grid-TransperantContent {
+    display: grid;
+    grid-template-columns: auto;
+    background: linear-gradient(transparent, rgba(1, 15, 29, 0.5), transparent, rgba(1, 15, 29, 0.5));
+
+    .TransparentP {
+        font-family: "Roboto Mono", monospace;
+        color: white;
+        font-size: 30px;
+        padding: 0 10px 0 10px;
```
Comment (akashdoppalapudi) [inline styles.scss]: padding: 0 10px;

File: styles.scss [hunk 13]
```diff
@@ -0,0 +1,201 @@
...
+    }
+}
+
+.MainContent {
+    display: grid;
+    grid-template-columns: 33.33% 33.33% 33.33%;
+    background-color: rgba(1, 15, 29, 0.8);
+    border: 1px solid rgba(23, 45, 69);
+    border-style: ridge;
+    outline-color: 0px;
```
Comment (akashdoppalapudi) [inline styles.scss]: color in pixels?

**Review by akashdoppalapudi — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (balaharikrishna) [reply index.html]: removed <br> tags and kept them in a Seperate <p>.

File: ourservicesstyles.css (hunk 3 above)
Comment (balaharikrishna) [reply ourservicesstyles.css]: changes applied.

File: ourservicesstyles.scss (hunk 5 above)
Comment (balaharikrishna) [reply ourservicesstyles.scss]: changes applied

File: styles.css (hunk 9 above)
Comment (balaharikrishna) [reply styles.css]: changes applied.

File: styles.scss (hunk 10 above)
Comment (balaharikrishna) [reply styles.scss]: changed to padding-bottom.

File: styles.scss (hunk 11 above)
Comment (balaharikrishna) [reply styles.scss]: applied the change :padding: 0 2%;

File: styles.scss (hunk 13 above)
Comment (balaharikrishna) [reply styles.scss]: removed it as it was not applicable and color can't be in pixels.

File: ourservicesstyles.scss (hunk 4 above)
Comment (balaharikrishna) [reply ourservicesstyles.scss]: applied the changes.

File: styles.scss (hunk 12 above)
Comment (balaharikrishna) [reply styles.scss]: applied the changes.

File: ourservicesstyles.css (hunk 2 above)
Comment (balaharikrishna) [reply ourservicesstyles.css]: applied the changes.

**Review by akashdoppalapudi — APPROVED** [review]

**Pull request merged**